src/binder/binder_context.cpp

```diff
@@ -18,9 +18,10 @@
 namespace peloton {
 namespace binder {
 
-void BinderContext::AddTable(const parser::TableRef* table_ref) {
+void BinderContext::AddTable(const parser::TableRef* table_ref,
+                             concurrency::Transaction* consistentTxn) {
   storage::DataTable* table = catalog::Catalog::GetInstance()->GetTableWithName(
-      table_ref->GetDatabaseName(), table_ref->GetTableName());
+      table_ref->GetDatabaseName(), table_ref->GetTableName(), consistentTxn);
```

Comment:
I use consistentTxn because I want to express this variables to keep txn handle consistent from "BEGIN" to "COMMIT"